navigation_and_routing/lib/src/app.dart

```diff
@@ -12,10 +12,10 @@ class Bookstore extends StatefulWidget {
   const Bookstore({Key? key}) : super(key: key);
 
   @override
-  _BookstoreState createState() => _BookstoreState();
+  BookstoreState createState() => BookstoreState();
```

Comment:
`State<Bookstore> createState() => _BookstoreState();`

Reply:
Fixed all instances in this PR